scripts/composer/ScriptHandler.php

```diff
@@ -14,8 +9,16 @@
 use DrupalFinder\DrupalFinderComposerRuntime;
 use Symfony\Component\Filesystem\Filesystem;
 
+/**
+ *
+ */
 class ScriptHandler {
 
+  /**
+   * @param \Composer\Script\Event $event
+   *
```

Comment:
@jonsimonsen I think it needs a proper description for that function + the class. An empty phpdoc comment does not makes much sense.

Reply:
Comments added by me.